### DISCUSSION TITLE
Lazar doesnt open anymore

### DISCUSSION THREAD

**Discussion by MCdipshit** [ppy/osu#19903] [Q&A] [answered]
### Type

Game behaviour

### Bug description

everytime i try to open lazar it says database migration failed even after updating and using the latest install 

### Screenshots or videos

![Desktop Screenshot 2022 08 21 - 18 34 34 53](https://user-images.githubusercontent.com/90268577/185815641-8b515254-2efa-43c6-bec9-77c127e784cf.png)

### Version

lazar

### Logs

[database.log](https://github.com/ppy/osu/files/9390174/database.log)
[network.log](https://github.com/ppy/osu/files/9390175/network.log)
[performance.log](https://github.com/ppy/osu/files/9390176/performance.log)
[performance-audio.log](https://github.com/ppy/osu/files/9390177/performance-audio.log)
[performance-draw.log](https://github.com/ppy/osu/files/9390178/performance-draw.log)
[performance-update.log](https://github.com/ppy/osu/files/9390179/performance-update.log)
[runtime.log](https://github.com/ppy/osu/files/9390180/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9390181/updater.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/9390182/legacy-ipc.log)

**Comment by ItsJustSomeDude** [comment]
I think you will have to upload all the `.realm` files from the `backups` folder in your osu! directory.

**Comment by MCdipshit** [comment]
Stable directory?

**Comment by peppy** [comment]
Could you start by taking a screenshot of your lazer folder (the one above where you found the logs)?

**Comment by MCdipshit** [comment]
![1](https://user-images.githubusercontent.com/90268577/185826185-d721b952-e03a-4c14-925f-a0646495e135.PNG)

**Comment by MCdipshit** [comment]
is that what you need?

**Comment by peppy** [comment]
Can you zip and attach `client.db`, `client.realm` and your `backups` folder please?

**Comment by MCdipshit** [comment]
i compressed backup into a zip file but it says its too large for github

**Comment by MCdipshit** [comment]
[client.zip](https://github.com/ppy/osu/files/9390767/client.zip)

**Comment by MCdipshit** [comment]
[client.zip](https://github.com/ppy/osu/files/9390771/client.zip)

**Comment by MCdipshit** [comment]
first is client.db second is client realm

**Comment by MCdipshit** [comment]
as stated previously backup folder is above the 65 mb limit

**Comment by MCdipshit** [comment]
[backups.zip](https://github.com/ppy/osu/files/9390796/backups.zip)

**Comment by MCdipshit** [comment]
thats all the realm files

**Reply by MCdipshit** [reply]
from backup that is

**Comment by peppy** [comment, marked as answer]
Okay so your previous database looks corrupt beyond recovery. Please delete `client.db` and reimport anything you care about.

**Reply by MCdipshit** [reply]
Example of items i should reimport are?

**Reply by MCdipshit** [reply]
Ah nevermind

**Reply by peppy** [reply]
Beatmaps etc. from stable, if you need that. Else just start from scratch.

**Reply by MCdipshit** [reply]
Thank you it works now
